src/table/TableWrapper.jsx

```diff
@@ -144,26 +151,36 @@ export default function TableWrapper(props) {
           <TableBodyWrapper {...props} setFocusedCellCoord={setFocusedCellCoord} setShouldRefocus={setShouldRefocus} />
         </Table>
       </TableContainer>
-      <TablePagination
-        className={classes[paginationHidden]}
-        rowsPerPageOptions={paginationFixedRpp ? [rowsPerPage] : [10, 25, 100]}
-        component="div"
-        count={size.qcy}
-        rowsPerPage={rowsPerPage}
-        labelRowsPerPage={`${translator.get('SNTable.Pagination.RowsPerPage')}:`}
-        page={page}
-        SelectProps={{
-          inputProps: {
-            'aria-label': translator.get('SNTable.Pagination.RowsPerPage'),
-            'data-testid': 'select',
-          },
-          native: true,
-        }}
-        labelDisplayedRows={({ from, to, count }) => `${from}-${to} of ${count}`}
-        onPageChange={handleChangePage}
-        onRowsPerPageChange={handleChangeRowsPerPage}
-        ActionsComponent={TablePaginationActions}
-      />
+      <Paper className={classes.tablePaginationSection}>
+        <TablePagination
+          className={classes[paginationHidden]}
+          rowsPerPageOptions={paginationFixedRpp ? [rowsPerPage] : [10, 25, 100]}
+          component="div"
+          count={size.qcy}
+          rowsPerPage={rowsPerPage}
+          labelRowsPerPage={`${translator.get('SNTable.Pagination.RowsPerPage')}:`}
+          page={page}
+          SelectProps={{
+            inputProps: {
+              'aria-label': translator.get('SNTable.Pagination.RowsPerPage'),
+              'data-testid': 'select',
+              tabindex: keyboard.active ? '0' : '-1',
+            },
+            native: true,
+          }}
+          labelDisplayedRows={({ from, to, count }) => `${from}-${to} of ${count}`}
+          onPageChange={handleChangePage}
+          onRowsPerPageChange={handleChangeRowsPerPage}
+          ActionsComponent={() => <div>{null}</div>}
+        />
+        <TablePaginationActions
+          count={size.qcy}
+          onPageChange={handleChangePage}
+          page={page}
+          rowsPerPage={rowsPerPage}
+          keyboard={keyboard}
+        />
```

Comment:
I think you can just `<TablePaginationActions {...props} keyboard={keyboard} />`

Reply:
size.qcy, handleChangePage, page, and rowsPerPage all are not in props.

Reply:
ahh, nvm. I mean the the props that `TablePaginationActions` gets if you just attach it as a function. When I tried this I changed it to `(props) => TablePaginationActions(..props, keyboard)`, but I didn't get that to memo properly